internal/coremsgs/en_api_translations.go

```diff
@@ -42,6 +42,7 @@ var (
 	APIParamsDatatypeVersion                = ffm("api.params.datatypeVersion", "The version of the datatype")
 	APIParamsEventID                        = ffm("api.params.eventID", "The event ID")
 	APIParamsFetchReferences                = ffm("api.params.fetchReferences", "When set, the API will return the record that this item references in its 'ref' field")
+	APIParamsFetchReference                 = ffm("api.params.fetchReference", "When set, the API will return the record that this item references in its 'ref' field")
```

Comment:
Kinda wish we'd used singular `fetchreference` across the board (instead of `fetchreferences`). Maybe we could honor both on the "get many" endpoint?